doc/setup/quickstart_cmake/index.md

```diff
@@ -102,33 +111,27 @@ To test whether everything works, we will now compile and run a small example.
 
 First we create the file `hello_world.cpp` in the `source` directory with the following contents:
 
-\include doc/setup/quickstart_cmake/hello_world.cpp
+\include test/external_project/src/hello_world.cpp
 
-To compile it we first create a `CMakeLists.txt` file in the `source` directory:
-
-```cmake
-cmake_minimum_required (VERSION 3.4)
-project (seqan3_tutorial CXX)
-
-find_package (SeqAn3 3.0.0 REQUIRED HINTS "${CMAKE_SOURCE_DIR}/../seqan3/build_system")
-
-add_executable (hello_world hello_world.cpp)
-
-target_link_libraries (hello_world seqan3::seqan3)
-```
+To compile it, we first create a `CMakeLists.txt` file in the `source` directory:
+<!-- Parsing the snippet like this to avoid verbatim includes of the snippet identifiers if we used nested snippets. -->
```

Comment:
```suggestion
<!-- snippet start: Parsing the snippet like this to avoid verbatim includes of the snippet identifiers if we used nested snippets. -->
```